CHANGES

```diff
@@ -17,6 +17,7 @@ Version 1.3.0 (released ??-???-????)
   * new 'root_path_locale' option (Tigris #500)
   * support Unicode root path both in CVS and Subversion (Tigris #500)
   * fix encoding problems on diff view and generating patch (#301)
+  * fix treat of symlink entries in tar ball (#344)
```

Comment:
"treatment".  But note that if this is not a bug in 1.2, we don't need to list it here at all.  The Version 1.3.0 notes are a _delta_ against a previous versions, not just a record of everything we did in this timeframe.

Reply:
I see.  I'll alter this commit without this change of CHANGES file.

Reply:
done.